session6/com.itschool.course/src/homework/Ex5.java

```diff
@@ -0,0 +1,20 @@
+package homework;
+
+public class Ex5 {
+
+    public static void main(String[] args) {
+
+        int evenCount = 0;
+        int oddCount = 0;
+
+        for(int i = 1; i <= 10; i++){
+            if(i % 2 == 0){
+                evenCount++;
```

Comment:
this will print the sum of even numbers, not their values. Instead incrementing this you should simply display them.